catalogs/catalog-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -352,6 +352,7 @@ public Schema createSchema(NameIdentifier ident, String comment, Map<String, Str
 
   @Override
   public Schema loadSchema(NameIdentifier ident) throws NoSuchSchemaException {
+    createDefaultSchemaIfNecessary();
```

Comment:
What is the purpose of moving to here, rather than doing this in initialization?

Reply:
Because the catalog entity was not created in the store during initialization, this will cause the default schema creation to fail. So I postponed the creation of default schema when operations

Reply:
So if users directly operate the topic like load topic, should you also guarantee that the default schema is already created, otherwise it will fail?

Reply:
> So if users directly operate the topic like load topic, should you also guarantee that the default schema is already created, otherwise it will fail?

It will not failed.
`checkSchemaExists` method will be called in front of `loadTopic` method, and  `createDefaultSchemaIfNecessary ` will be called in `checkSchemaExists`

Reply:
> Seems only when calling schema-related APIs will you creates a "default" schema

No, topic-related operations will call `checkSchemaExists`, it will `createDefaultSchemaIfNecessary`

Reply:
Currently, each schema related operations will need to check and create a default schema, maybe we can do this in a unified place as we discussed offline, let's do this in a separate PR.

Reply:
OK, Will solve this problem together when solving #2795